packages/material-ui-lab/src/SliderStyled/SliderStyled.d.ts

```diff
@@ -48,4 +48,46 @@ export const SliderValueLabel: React.FC<SliderValueLabel>;
  */
 declare const Slider: ExtendSliderUnstyled<SliderStyledTypeMap>;
 
+export type SliderStyledClassKey =
+  /** Styles applied to the root element. */
+  | 'root'
+  /** Styles applied to the root element if `color="primary"`. */
+  | 'colorPrimary'
+  /** Styles applied to the root element if `color="secondary"`. */
+  | 'colorSecondary'
+  /** Styles applied to the root element if `marks` is provided with at least one label. */
+  | 'marked'
+  /** Pseudo-class applied to the root element if `orientation="vertical"`. */
+  | 'vertical'
+  /** Pseudo-class applied to the root and thumb element if `disabled={true}`. */
+  | 'disabled'
+  /** Styles applied to the rail element. */
+  | 'rail'
+  /** Styles applied to the track element. */
+  | 'track'
+  /** Styles applied to the track element if `track={false}`. */
+  | 'trackFalse'
+  /** Styles applied to the track element if `track="inverted"`. */
+  | 'trackInverted'
+  /** Styles applied to the thumb element. */
+  | 'thumb'
+  /** Styles applied to the thumb element if `color="primary"`. */
+  | 'thumbColorPrimary'
```

Comment:
Does this value still make sense? Are we still generating a class name for it?

Reply:
Not sure if it makes sense, but we are still generating className for it https://github.com/mui-org/material-ui/blob/next/packages/material-ui-lab/src/SliderStyled/SliderStyled.js#L49